diff --git a/src/cli.js b/src/cli.js
index d3eae6b4fd01cd801eb6574ee90dec33e2eb6c62..dc67aeb207665763e34be3831f2aba93ecb1baac 100644
--- a/src/cli.js
+++ b/src/cli.js
@@ -14,7 +14,7 @@ function handle_error(e) {
 
 	console.error(styleText(['bold', 'red'], `> ${error.message}`));
 	if (error.stack) {
-		console.error(styleText('grey', error.stack.split('\n').slice(1).join('\n')));
+		console.error(styleText('gray', error.stack.split('\n').slice(1).join('\n')));
 	}
 
 	process.exit(1);
diff --git a/src/core/utils.js b/src/core/utils.js
index e2be24be01d74608764773755795e378173330df..13a6d3c3df2ce788684a15980ecf8929a0af27e8 100644
--- a/src/core/utils.js
+++ b/src/core/utils.js
@@ -49,7 +49,7 @@ export function logger({ verbose } = { verbose: true }) {
 	log.success = (msg) => log(styleText('green', `✔ ${msg}`));
 	log.error = (msg) => err(styleText(['bold', 'red'], msg));
 	log.warn = (msg) => log(styleText(['bold', 'yellow'], msg));
-	log.minor = verbose ? (msg) => log(styleText('grey', msg)) : noop;
+	log.minor = verbose ? (msg) => log(styleText('gray', msg)) : noop;
 	log.info = verbose ? log : noop;
 	log.err = err;
 
